README.md

```diff
@@ -17,21 +17,20 @@ In order to deploy the app in this repo you need to follow these steps:
 - Add a virtual hosts entry pointing at the directory where you extracted the
   files in the previous step. Make sure that the installation is in the
   document root.
+- Run `composer install` to download and configure dependencies and library files
 - Configure the web server (see below)
 - Create the database for the server
 - install mbstring (required for cake 3.0)
-- install intl extension: `sudo apt-get install php5-intl` //(required for cake 3.0)
+- install intl extension; on Debian use: `sudo apt-get install php-intl` //(required for cake 3.0)
 - cd application_root_dir (directory under which subdirectory `src` resides)
 - mkdir tmp;
 - mkdir logs;
 - set permissions for tmp and logs directory
-          ``
-	- ```HTTPDUSER=`ps aux | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | head -1 | cut -d\  -f1````
+	-     HTTPDUSER=`ps aux | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | head -1 | cut -d\  -f1`
```

Comment:
IMHO this will look wrong - it should be rendered as 
```
HTTPDUSER=`ps aux...`
```
Not as
HTTPDUSER=`ps aux...`
(The latter one is missing backticks around the statement)

Reply:
I believe it's rendered correctly through GitHub markdown, I've tested at https://github.com/ibennetch/test/wiki/Bullet-point-commands

It's still a little ugly, and it's not rendered quite the same as the backtick-markdown, but I don't know another way to show both spaces after the backslash (without both spaces the cut command fails).

Reply:
Okay, you're right it seems correct (but still a confusing syntax).